### ISSUE TITLE
Automatic unequipping readds incorrect item and strips its data values

### ISSUE THREAD

**Issue by estralis** [Illarion-eV/Illarion-Content#319]
[https://github.com/Illarion-eV/Illarion-Content/blob/master/base/fightingutil.lua#L80](url)

Problems with this code: The code just checks if any weapon is broken and then by default removes the one in the right hand even if the one in the left is actually broken. Also, it looks like the wrong type of struct is used to recreate the item; the weapon struct is generic while we need the specific script item.

Also, I'd propose to move the logics of unequiping weapons to server.standardfighting where the unequiping of armour is handled.

**Comment by estralis** [comment]
When fixing this, all kinds of combinations of broken weapons, arrows, shields, tools, fruits,... in hands should be tested in both slots.

**Comment by Brightrim** [comment]
Pushed a solution to this. Since item.data is not a thing and we have no other way to get the complete data table of an item, the M.readdItem(user, item) function will always erase the data values of an item. I created a work around instead. 

While I already tested it personally, it should ideally be tested by other people as well before being pushed to live. 

It also requires one line to be translated to german, marked with "GERMAN TRANSLATION" in the common script. 

I'd argue that once this is complete, it should be put in as a hotfix instead of being part of the next update milestone(as it seems to have been added to above), seeing how it is quite game breaking that peoples items being broken can lead to them losing things like their magical gems or items with special descriptions and the likes, as it is currently.
